### PR TITLE
code: add traditional Chinese translation

### PR DESCRIPTION
**Pull request by blueskyson**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh_TW/common/code.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# code
+
+> Visual Studio Code.
```
Comment (CleanMachine1) [inline pages.zh_TW/common/code.md:3]: Can his not be translated

Comment (CleanMachine1) [reply pages.zh_TW/common/code.md:3]: Does the Chinese version use a word instead of Visual Studio Code

Comment (blueskyson) [reply pages.zh_TW/common/code.md:3]: I've hardly ever seen the words "visual studio code" written in Chinese. I can add the Chinese translation, but we usually mention it in English words.

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @blueskyson

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**